finality-aleph/src/abft/current.rs

```diff
@@ -76,11 +79,16 @@ pub fn create_aleph_config(
     unit_creation_delay: UnitCreationDelay,
 ) -> Config {
     let delay_config = DelayConfig {
-        tick_interval: Duration::from_millis(100),
+        tick_interval: Duration::from_millis(10),
```

Comment:
why 100 -> 10?

Reply:
This and all changes in config are consequences of changes in aleph-bft